unified_targets/configs/MANGOF722.config

```diff
@@ -0,0 +1,125 @@
+# Betaflight / MANGOF722 (MGF7) 4.0.3 Jun 26 2019 / 10:28:42 () MSP API: 1.41
```

Comment:
Should be:
`# Betaflight / STM32F7X2 (S7X2) 4.0.3 Jun 26 2019 / 10:28:42 () MSP API: 1.41`

As part of implementing unified targets, the CPU type needs to be recorded somewhere.